packages/rln/src/utils/epoch.ts

```diff
@@ -15,11 +17,7 @@ export function dateToEpoch(
 }
 
 export function epochIntToBytes(epoch: number): Uint8Array {
-  const bytes = new Uint8Array(32);
-  const db = new DataView(bytes.buffer);
-  db.setUint32(0, epoch, true);
-  log.info("encoded epoch", epoch, bytes);
-  return bytes;
+  return BytesUtils.writeUIntLE(new Uint8Array(32), epoch, 0, 32);
```

Comment:
nit: to keep compatible with JS definitions it should be `writeUintLE` instead of `writeUIntLE`